### DISCUSSION TITLE
Unable to open. ipynb file

### DISCUSSION THREAD

**Discussion by PoworMan** [coder/code-server#6296] [Q&A]
Python and jupyter plugins have been installed.
Always loading ...

**Comment by lg-nineoneninethreenine** [comment]
Well, I have noticed a constant issue with Python and Jupyter. Can you just check the issues/discussions with the query "Jupyter"? I think there was an issue where a fix was suggested. I am unfortunately kinda lazy right now because it is bedtime for me, but try searching at "Issues" and "Discussions", it _could_ help. If you need more help, try posting details here.
And if you can, please post a screenshot so that others can also help out if they recognise it.

**Reply by lg-nineoneninethreenine** [reply]
/ping @PoworMan

Need you help on this!
